extensions/currency-exchange/src/index.tsx

```diff
@@ -226,7 +226,6 @@ function useExchange() {
         isLoading: true,
       }));
       try {
-        console.log("country:", state.fromCurrencyCode);
```

Comment:
What's wrong with this conole.log line?

Reply:
I think it is just the a log for debugging, and it indeed should be deleted when publishing to production.